HARMONIZED_TEST_ENVIRONMENT.md

```diff
@@ -0,0 +1,72 @@
+Harmonized Xroad test environment
+===================
+
+- Common test development environment to Finland and Estonia
+	- Includes servers: CA, CENTRAL SERVER, MANAGEMENT SECURITY SERVER, SECURITY SERVERS
+- Can be used for testing:
+	- UI and integrations, SOAP testing, Service testing and Service mocks
+	- Not for performance testing!
+- Benefits:
+	- Tests are compatible in Estonia and Finland common test environments
+		- Re-usable tests
+	- Test can be easily imported to own specific environment from LXD
+	- Jenkins jobs are generated and we have harmonized test data in different test environments
+	- LXD will easy up environment settings work
```

Comment:
Rephrase to something like "Less work setting up environment"

Reply:
done